crypto/test_pairs.ts

```diff
@@ -30,3 +32,36 @@ export const bobStash = pair(
 function pair(secret: string) {
   return Sr25519.fromSecret(decode(secret))
 }
+
+export function testUser(userId: number) {
+  return Sr25519.fromSeed(
+    Blake2_256.hash(new TextEncoder().encode(`capi-test-user-${userId}`)),
+  )
+}
+
+export function testUserFactory(url: string) {
+  return async function users<N extends number>(count: N): Promise<ArrayOfLength<Sr25519, N>> {
+    const response = await fetch(
+      url,
+      {
+        method: "POST",
+        headers: {
+          "content-type": "application/json",
+        },
+        body: JSON.stringify({ count }),
```

Comment:
Why does the count need to be contained within an object? Wanna just have the count be the body?

Reply:
If we eventually need to add more data, I think that it's easier to augment the body with an extra attribute.
The above assumes that the CAPI server could eventually be deployed as a cloud service.

If that assumption is not accurate, then, this and the response could be raw text.

Shall I do the switch to raw text?

Reply:
for a backward compatible change.
For example, the client code is cached by the deno cache and the server is updated keeping the old object attribute and adding a new one